src/DependencyInjection/DoctrineExtension.php

```diff
@@ -418,8 +418,8 @@ private function detectMappingType(string $directory, ContainerBuilder $containe
             }
 
             if (
-                preg_match('/^(?: \*|\/\*\*) @.*' . $quotedMappingObjectName . '\b/m', $content)
-                || preg_match('/^(?: \*|\/\*\*) @.*Embeddable\b/m', $content)
+                preg_match('/^(?: \*|\/\*\*) @[a-zA-Z\\\\]*' . $quotedMappingObjectName . '\b/m', $content)
```

Comment:
`a-zA-Z` is not the range of characters allowed in PHP namespaces. Your current regex will reject some valid class names.